packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapCircleController.m

```diff
@@ -99,6 +108,11 @@ - (void)interpretCircleOptions:(NSDictionary *)data {
   if (fillColor) {
     [self setFillColor:[FLTGoogleMapJSONConversions colorFromRGBA:fillColor]];
   }
+
+  NSNumber *visible = FGMGetValueOrNilFromDict(data, @"visible");
+  if (visible) {
+    [self setVisible:[visible boolValue]];
+  }
```

Comment:
Moving this code doesn't seem relevant to your change. Please revert to minimise diffs.

Reply:
Hi @cbracken, have you missed this reply?

Reply:
If the order of this call relative to everything else in the method is critical, please add a comment to that effect on this part of the code, otherwise it's not going to be clear to people editing (or reviewing changes to) these files in the future that adding new properties after this block is incorrect.

Reply:
Okay I added a comment to each of the instances of this change